src/greynirseq/nicenlp/utils/logits_filter.py

```diff
@@ -73,7 +65,7 @@ def max_tensor_by_bins(tensor, bins, softmax_by_bin=False):
             if bin_end - bin_start > 0:
                 bin_tensors = F.softmax(bin_tensors)
             else:
-                bin_tensors = F.sigmoid(bin_tensors)
+                bin_tensors = torch.sigmoid(bin_tensors)
```

Comment:
Any reason for this? I understand they are more or less the same but we'd rather not change something as sensitive unless there is a good reason

Reply:
There was a deprecation warning, so I changed it.
The current implementation in PyTorch for `F.sigmoid` is: `input.sigmoid()`. I would expect `torch.sigmoid` to behave in the same way.